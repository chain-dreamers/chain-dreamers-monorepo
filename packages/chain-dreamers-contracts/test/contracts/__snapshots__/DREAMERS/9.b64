data:application/json;base64,ZGF0YTphcHBsaWNhdGlvbi9qc29uLHsiaW1hZ2VfZGF0YSI6ICIlM2NzdmclMjB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnJTIwdmlld0JveD0nMCUyMDAlMjAyNTUlMjAyNTUnJTIwd2lkdGg9JzUwMHB4JyUyMGhlaWdodD0nNTAwcHgnJTNlJTNjcGF0aCUyMGQ9J00wLDBIMjU1VjI1NUgwVjAnJTIwZmlsbD0nJTIzNkUyQUY1JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE3MywyMTdIODJDNTQsMjI0LDQ4LDI1NSw0OCwyNTVIMjA3QzIwNywyNTUsMjAxLDIyNCwxNzMsMjE3JyUyMGZpbGw9JyUyMzZFODFBRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J004MiwyMTFIMTczVjI1NUg4MlYyMTEnJTIwZmlsbD0nJTIzRThDNDlGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTgzLDIxMVYyMjVDOTYsMjM4LDExMSwyNDYsMTI3LDI0NkMxNDQsMjQ2LDE1OSwyMzgsMTcyLDIyNVYyMTFIODMnJTIwZmlsbD0nJTIzRENBQjdFJy8lM2UlM2NwYXRoJTIwZD0nTTE5NywxMDZINThDNDYsMTA2LDM2LDExNywzNiwxMzFDMzYsMTQ1LDQ2LDE1Niw1OCwxNTZIMTk3QzIwOSwxNTYsMjE5LDE0NSwyMTksMTMxQzIxOSwxMTcsMjA5LDEwNiwxOTcsMTA2JyUyMGZpbGw9JyUyM0QxQTE2RiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001OCwxMzVBNjksOTksMCwxLDAsMTk3LDEzNUE2OSw5OSwwLDEsMCw1OCwxMzUnJTIwZmlsbD0nJTIzRThDNDlGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE5NiwxMjBDMTk2LDEyMCwyMDksMTE4LDIxMCwxMzUnJTIwZmlsbD0nbm9uZSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001OSwxMjBDNTksMTIwLDQ2LDExOCw0NSwxMzUnJTIwZmlsbD0nbm9uZSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J003OSwxMzVDNzksODYsMTA0LDQ1LDEzNiwzOEE0NCw0NCwwLDAsMCwxMjYsMzdDODksMzcsNTksODEsNTksMTM1QzU5LDE5MCw4OSwyMzMsMTI2LDIzM0E1MCw1MCwwLDAsMCwxMzYsMjMyQzEwNCwyMjUsNzksMTg1LDc5LDEzNSclMjBmaWxsPSclMjNEQ0FCN0UnLyUzZSUzY3BhdGglMjBkPSdNMTI3LDE5NkMxMTEsMTk2LDEwNywxOTIsMTA3LDE5MkMxMDcsMTkyLDExMywyMDYsMTI3LDIwNkMxNDIsMjA2LDE0OCwxOTIsMTQ4LDE5MkMxNDgsMTkyLDE0NCwxOTYsMTI3LDE5NiclMjBmaWxsPSclMjMxNzE3MTcnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTMxLDE3NEgxMjNBNCw0LDAsMCwwLDExOSwxNzlWMTc5QTQsNCwwLDAsMCwxMjMsMTg0SDEzMUE0LDQsMCwwLDAsMTM2LDE3OVYxNzlBNCw0LDAsMCwwLDEzMSwxNzQnJTIwZmlsbD0nJTIzMUUxRTFFJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTc2LDEzNkEyMCwxNiwwLDEsMCwxMTYsMTM2QTIwLDE2LDAsMSwwLDc2LDEzNiclMjBmaWxsPSclMjNDQjVGNjYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTA3LDEyOUEyOCwyOCwwLDAsMCwxMDMsMTI5Qzk0LDEyOSw4NiwxMzMsODYsMTM5Qzg2LDE0NCw5NCwxNDksMTAzLDE0OUEyOCwyOCwwLDAsMCwxMDcsMTQ4QzEwMCwxNDcsOTQsMTQzLDk0LDEzOUM5NCwxMzQsMTAwLDEzMCwxMDcsMTI5JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNDMsMTM5QzE0MywxMzQsMTQ4LDEzMCwxNTYsMTI5QTI4LDI4LDAsMCwwLDE1MiwxMjlDMTQyLDEyOSwxMzUsMTMzLDEzNSwxMzlDMTM1LDE0NCwxNDIsMTQ5LDE1MiwxNDlBMjgsMjgsMCwwLDAsMTU2LDE0OEMxNDgsMTQ3LDE0MywxNDMsMTQzLDEzOSclMjBmaWxsPSclMjMwMDAwMDAnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTA3LDEyOUMxMDAsMTMwLDk0LDEzNCw5NCwxMzlDOTQsMTQzLDEwMCwxNDcsMTA3LDE0OEMxMTUsMTQ3LDEyMCwxNDMsMTIwLDEzOUMxMjAsMTM0LDExNSwxMzAsMTA3LDEyOSclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTU2LDEyOUMxNDgsMTMwLDE0MywxMzQsMTQzLDEzOUMxNDMsMTQzLDE0OCwxNDcsMTU2LDE0OEMxNjMsMTQ3LDE2OSwxNDMsMTY5LDEzOUMxNjksMTM0LDE2MywxMzAsMTU2LDEyOSclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNODcsMTA1QTYsNiwwLDAsMCw4MywxMTZBNiw2LDAsMCwwLDkwLDExOEM5NiwxMTYsMTA4LDExNSwxMTcsMTI4QzExNywxMjgsMTE1LDEwMSw4NywxMDUnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE2OCwxMDVDMTQwLDEwMSwxMzgsMTI4LDEzOCwxMjhDMTQ3LDExNSwxNTksMTE2LDE2NSwxMThBNiw2LDAsMCwwLDE3MiwxMTZBNiw2LDAsMCwwLDE2OCwxMDUnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwNywxMzVBMywzLDAsMCwwLDEwNCwxMzlBMywzLDAsMCwwLDEwNywxNDJBMywzLDAsMCwwLDExMSwxMzlBMywzLDAsMCwwLDEwNywxMzUnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE1NiwxMzVBMywzLDAsMCwwLDE1MiwxMzlBMywzLDAsMCwwLDE1NiwxNDJBMywzLDAsMCwwLDE1OSwxMzlBMywzLDAsMCwwLDE1NiwxMzUnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTk4LDg3QzExMiw4NywxMDksOTgsMTI0LDk1QzE0MCw5MywxNDUsODEsMTU3LDkxJyUyMGZpbGw9J25vbmUnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTEwLDEwMUMxMTgsMTAxLDExNiwxMDcsMTI2LDEwNkMxMzUsMTA0LDEzOCw5NywxNDUsMTAzJyUyMGZpbGw9J25vbmUnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNODAsMTM5QTIzLDE1LDAsMSwwLDEyNiwxMzlBMjMsMTUsMCwxLDAsODAsMTM5JyUyMGZpbGw9JyUyM0ZGRkZGRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMjksMTM5QTIzLDE1LDAsMSwwLDE3NSwxMzlBMjMsMTUsMCwxLDAsMTI5LDEzOSclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMjAyLDE1M0E1LDUsMCwwLDAsMTk3LDE1OUE1LDUsMCwxLDAsMjAyLDE1MyclMjBmaWxsPSclMjNBQkFCQUInJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNNTMsMTUzQTUsNSwwLDAsMCw0NywxNTlBNSw1LDAsMSwwLDUzLDE1MyclMjBmaWxsPSclMjNBQkFCQUInJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMjAzLDE1NEE1LDUsMCwwLDAsMTk5LDE1NVYxNTVBNSw1LDAsMCwxLDIwNCwxNjBBNCw0LDAsMCwxLDIwMywxNjRBNSw1LDAsMSwwLDIwMywxNTRWMTU0JyUyMGZpbGw9JyUyMzc3Nzc3NycvJTNlJTNjcGF0aCUyMGQ9J000NywxNTlBNSw1LDAsMCwwLDUyLDE2NEE0LDQsMCwwLDEsNTEsMTYwQTUsNSwwLDAsMSw1NiwxNTVWMTU1QTQsNCwwLDAsMCw1MiwxNTRBNSw1LDAsMCwwLDQ3LDE1OSclMjBmaWxsPSclMjM3Nzc3NzcnLyUzZSUzY3BhdGglMjBkPSdNMTczLDIxN0g4MkM1NCwyMjQsNDgsMjU1LDQ4LDI1NUgyMDdDMjA3LDI1NSwyMDEsMjI0LDE3MywyMTcnJTIwZmlsbD0nJTIzMDYwNTA4JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTgyLDIxMUgxNzNWMjU1SDgyVjIxMSclMjBmaWxsPSclMjMyMjIwMzMnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNODIsMjExVjIyNUM5NSwyMzgsMTExLDI0NiwxMjcsMjQ2QzE0NCwyNDYsMTYwLDIzOCwxNzMsMjI1VjIxMUg4MiclMjBmaWxsPSclMjMwMDAwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTkzLDE0NEg2MkE2LDYsMCwwLDAsNTUsMTUyQzYzLDE5OSw5MiwyMzUsMTI3LDIzNUMxNjMsMjM1LDE5MiwxOTksMTk5LDE1MkE2LDYsMCwwLDAsMTkzLDE0NCclMjBmaWxsPSclMjMyMjIwMzMnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTkzLDE0NEgxNTNDMTQ2LDE4NCwxMjQsMjE2LDk2LDIyNUMxMDUsMjMxLDExNiwyMzUsMTI3LDIzNUMxNjMsMjM1LDE5MiwxOTksMTk5LDE1MkE2LDYsMCwwLDAsMTkzLDE0NCclMjBmaWxsPSclMjMwMDAwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTY2LDQyQTQyLDQyLDAsMCwxLDE2Miw1NUMxNzEsNjEsMTk3LDk0LDE5NywxMjVDMTk3LDEyNSwyMTEsNDYsMTY2LDQyJyUyMGZpbGw9JyUyMzUxM0UzQSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J002MSw4NEMzMywxMjQsNjEsMTc5LDYxLDE3OUM2MSwyMjEsOTYsMjQyLDk2LDI0MkM4MiwxNDUsOTYsOTAsOTYsOTBDMTA0LDkxLDExMiw4NywxMTMsNzZDMTQzLDgwLDE1Niw2NiwxNjIsNTVBNDIsNDIsMCwwLDAsMTY2LDQyQzEwMywwLDU2LDYwLDYxLDg0JyUyMGZpbGw9JyUyMzc2NEI0MiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMzksMzJBNjYsNjYsMCwwLDEsMTQ5LDMzQTY4LDY4LDAsMCwwLDEyMywyOEMxMDQsMjgsODYsMzcsNzMsNTJDNjQsNjMsNjAsNzYsNjEsODRWODRWODRDMzMsMTI0LDYxLDE3OCw2MSwxNzlWMTc5VjE3OUM2MSwyMTYsOTAsMjM3LDk1LDI0MEM5NSwyMzcsOTQsMjM0LDk0LDIzMUM4MywyMTcsNzcsMjAyLDc3LDE4NUM3NiwxODIsNTAsMTI5LDc3LDkwQzc1LDgxLDgwLDY4LDg5LDU3QzEwMiw0MiwxMjAsMzIsMTM5LDMyJyUyMGZpbGw9JyUyMzQ3MjYyMCcvJTNlJTNjcGF0aCUyMGQ9J00xODcsNTRBMjgsMjgsMCwwLDAsMTY2LDQzQTQ2LDQ2LDAsMCwxLDE2NCw1MUMxODcsNTUsMTkzLDgyLDE5MywxMDNBNzksNzksMCwwLDEsMTk3LDEyMEMxOTksMTA3LDIwMSw3MiwxODcsNTQnJTIwZmlsbD0nJTIzMzAyMTFGJy8lM2UlM2NzdHlsZSUzZXBhdGh7c3Ryb2tlLXdpZHRoOjAuNzF9JTNjL3N0eWxlJTNlJTNjL3N2ZyUzZSIsICJuYW1lIjogIkRyZWFtZXIlMjAlMjM5IiwgImRlc2NyaXB0aW9uIjogIlJ1bm5lcnMlMjBydW4sJTIwYnV0JTIwc29tZXRpbWVzJTIwdGhleSUyMGRyZWFtLiUyMFRoaXMlMjBpcyUyMG9uZSUyMG9mJTIwdGhlaXIlMjBkcmVhbXMuIiwiYXR0cmlidXRlcyI6ICIifQ==