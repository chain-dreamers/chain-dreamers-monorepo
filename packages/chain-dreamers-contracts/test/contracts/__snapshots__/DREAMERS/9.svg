data:image/svg+xml,%3csvg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20255%20255'%20width='500px'%20height='500px'%3e%3cpath%20d='M0%200H255V255H0V0'%20fill='%236E2AF5'%20stroke='%23000'/%3e%3cpath%20d='M173%20217H82C54%20224%2048%20255%2048%20255H207C207%20255%20201%20224%20173%20217'%20fill='%236E81AF'%20stroke='%23000'/%3e%3cpath%20d='M82%20211H173V255H82V211'%20fill='%23E8C49F'%20stroke='%23000'/%3e%3cpath%20d='M83%20211V225C96%20238%20111%20246%20127%20246C144%20246%20159%20238%20172%20225V211H83'%20fill='%23DCAB7E'/%3e%3cpath%20d='M197%20106H58C46%20106%2036%20117%2036%20131C36%20145%2046%20156%2058%20156H197C209%20156%20219%20145%20219%20131C219%20117%20209%20106%20197%20106'%20fill='%23D1A16F'%20stroke='%23000'/%3e%3cpath%20d='M58%20135A69%2099%200%201%200%20197%20135A69%2099%200%201%200%2058%20135'%20fill='%23E8C49F'%20stroke='%23000'/%3e%3cpath%20d='M196%20120C196%20120%20209%20118%20210%20135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M59%20120C59%20120%2046%20118%2045%20135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M79%20135C79%2086%20104%2045%20136%2038A44%2044%200%200%200%20126%2037C89%2037%2059%2081%2059%20135C59%20190%2089%20233%20126%20233A50%2050%200%200%200%20136%20232C104%20225%2079%20185%2079%20135'%20fill='%23DCAB7E'/%3e%3cpath%20d='M127%20196C111%20196%20107%20192%20107%20192C107%20192%20113%20206%20127%20206C142%20206%20148%20192%20148%20192C148%20192%20144%20196%20127%20196'%20fill='%23171717'%20stroke='%23000'/%3e%3cpath%20d='M131%20174H123A4%204%200%200%200%20119%20179V179A4%204%200%200%200%20123%20184H131A4%204%200%200%200%20136%20179V179A4%204%200%200%200%20131%20174'%20fill='%231E1E1E'%20stroke='%23000'/%3e%3cpath%20d='M76%20136A20%2016%200%201%200%20116%20136A20%2016%200%201%200%2076%20136'%20fill='%23CB5F66'%20stroke='%23000'/%3e%3cpath%20d='M107%20129A28%2028%200%200%200%20103%20129C94%20129%2086%20133%2086%20139C86%20144%2094%20149%20103%20149A28%2028%200%200%200%20107%20148C100%20147%2094%20143%2094%20139C94%20134%20100%20130%20107%20129'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M143%20139C143%20134%20148%20130%20156%20129A28%2028%200%200%200%20152%20129C142%20129%20135%20133%20135%20139C135%20144%20142%20149%20152%20149A28%2028%200%200%200%20156%20148C148%20147%20143%20143%20143%20139'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M107%20129C100%20130%2094%20134%2094%20139C94%20143%20100%20147%20107%20148C115%20147%20120%20143%20120%20139C120%20134%20115%20130%20107%20129'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M156%20129C148%20130%20143%20134%20143%20139C143%20143%20148%20147%20156%20148C163%20147%20169%20143%20169%20139C169%20134%20163%20130%20156%20129'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M87%20105A6%206%200%200%200%2083%20116A6%206%200%200%200%2090%20118C96%20116%20108%20115%20117%20128C117%20128%20115%20101%2087%20105'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M168%20105C140%20101%20138%20128%20138%20128C147%20115%20159%20116%20165%20118A6%206%200%200%200%20172%20116A6%206%200%200%200%20168%20105'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M107%20135A3%203%200%200%200%20104%20139A3%203%200%200%200%20107%20142A3%203%200%200%200%20111%20139A3%203%200%200%200%20107%20135'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M156%20135A3%203%200%200%200%20152%20139A3%203%200%200%200%20156%20142A3%203%200%200%200%20159%20139A3%203%200%200%200%20156%20135'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M98%2087C112%2087%20109%2098%20124%2095C140%2093%20145%2081%20157%2091'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M110%20101C118%20101%20116%20107%20126%20106C135%20104%20138%2097%20145%20103'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M80%20139A23%2015%200%201%200%20126%20139A23%2015%200%201%200%2080%20139'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M129%20139A23%2015%200%201%200%20175%20139A23%2015%200%201%200%20129%20139'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M202%20153A5%205%200%200%200%20197%20159A5%205%200%201%200%20202%20153'%20fill='%23ABABAB'%20stroke='%23000'/%3e%3cpath%20d='M53%20153A5%205%200%200%200%2047%20159A5%205%200%201%200%2053%20153'%20fill='%23ABABAB'%20stroke='%23000'/%3e%3cpath%20d='M203%20154A5%205%200%200%200%20199%20155V155A5%205%200%200%201%20204%20160A4%204%200%200%201%20203%20164A5%205%200%201%200%20203%20154V154'%20fill='%23777777'/%3e%3cpath%20d='M47%20159A5%205%200%200%200%2052%20164A4%204%200%200%201%2051%20160A5%205%200%200%201%2056%20155V155A4%204%200%200%200%2052%20154A5%205%200%200%200%2047%20159'%20fill='%23777777'/%3e%3cpath%20d='M173%20217H82C54%20224%2048%20255%2048%20255H207C207%20255%20201%20224%20173%20217'%20fill='%23060508'%20stroke='%23000'/%3e%3cpath%20d='M82%20211H173V255H82V211'%20fill='%23222033'%20stroke='%23000'/%3e%3cpath%20d='M82%20211V225C95%20238%20111%20246%20127%20246C144%20246%20160%20238%20173%20225V211H82'%20fill='%23000000'/%3e%3cpath%20d='M193%20144H62A6%206%200%200%200%2055%20152C63%20199%2092%20235%20127%20235C163%20235%20192%20199%20199%20152A6%206%200%200%200%20193%20144'%20fill='%23222033'%20stroke='%23000'/%3e%3cpath%20d='M193%20144H153C146%20184%20124%20216%2096%20225C105%20231%20116%20235%20127%20235C163%20235%20192%20199%20199%20152A6%206%200%200%200%20193%20144'%20fill='%23000000'/%3e%3cpath%20d='M166%2042A42%2042%200%200%201%20162%2055C171%2061%20197%2094%20197%20125C197%20125%20211%2046%20166%2042'%20fill='%23513E3A'%20stroke='%23000'/%3e%3cpath%20d='M61%2084C33%20124%2061%20179%2061%20179C61%20221%2096%20242%2096%20242C82%20145%2096%2090%2096%2090C104%2091%20112%2087%20113%2076C143%2080%20156%2066%20162%2055A42%2042%200%200%200%20166%2042C103%200%2056%2060%2061%2084'%20fill='%23764B42'%20stroke='%23000'/%3e%3cpath%20d='M139%2032A66%2066%200%200%201%20149%2033A68%2068%200%200%200%20123%2028C104%2028%2086%2037%2073%2052C64%2063%2060%2076%2061%2084V84V84C33%20124%2061%20178%2061%20179V179V179C61%20216%2090%20237%2095%20240C95%20237%2094%20234%2094%20231C83%20217%2077%20202%2077%20185C76%20182%2050%20129%2077%2090C75%2081%2080%2068%2089%2057C102%2042%20120%2032%20139%2032'%20fill='%23472620'/%3e%3cpath%20d='M187%2054A28%2028%200%200%200%20166%2043A46%2046%200%200%201%20164%2051C187%2055%20193%2082%20193%20103A79%2079%200%200%201%20197%20120C199%20107%20201%2072%20187%2054'%20fill='%2330211F'/%3e%3cstyle%3epath{stroke-width:0.71}%3c/style%3e%3c/svg%3e