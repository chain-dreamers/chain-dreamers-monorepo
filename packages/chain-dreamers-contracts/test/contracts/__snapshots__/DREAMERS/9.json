data:application/json,{"image_data": "%3csvg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20255%20255'%20width='500px'%20height='500px'%3e%3cpath%20d='M0,0H255V255H0V0'%20fill='%236E2AF5'%20stroke='%23000'/%3e%3cpath%20d='M173,217H82C54,224,48,255,48,255H207C207,255,201,224,173,217'%20fill='%236E81AF'%20stroke='%23000'/%3e%3cpath%20d='M82,211H173V255H82V211'%20fill='%23E8C49F'%20stroke='%23000'/%3e%3cpath%20d='M83,211V225C96,238,111,246,127,246C144,246,159,238,172,225V211H83'%20fill='%23DCAB7E'/%3e%3cpath%20d='M197,106H58C46,106,36,117,36,131C36,145,46,156,58,156H197C209,156,219,145,219,131C219,117,209,106,197,106'%20fill='%23D1A16F'%20stroke='%23000'/%3e%3cpath%20d='M58,135A69,99,0,1,0,197,135A69,99,0,1,0,58,135'%20fill='%23E8C49F'%20stroke='%23000'/%3e%3cpath%20d='M196,120C196,120,209,118,210,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M59,120C59,120,46,118,45,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M79,135C79,86,104,45,136,38A44,44,0,0,0,126,37C89,37,59,81,59,135C59,190,89,233,126,233A50,50,0,0,0,136,232C104,225,79,185,79,135'%20fill='%23DCAB7E'/%3e%3cpath%20d='M127,196C111,196,107,192,107,192C107,192,113,206,127,206C142,206,148,192,148,192C148,192,144,196,127,196'%20fill='%23171717'%20stroke='%23000'/%3e%3cpath%20d='M131,174H123A4,4,0,0,0,119,179V179A4,4,0,0,0,123,184H131A4,4,0,0,0,136,179V179A4,4,0,0,0,131,174'%20fill='%231E1E1E'%20stroke='%23000'/%3e%3cpath%20d='M76,136A20,16,0,1,0,116,136A20,16,0,1,0,76,136'%20fill='%23CB5F66'%20stroke='%23000'/%3e%3cpath%20d='M107,129A28,28,0,0,0,103,129C94,129,86,133,86,139C86,144,94,149,103,149A28,28,0,0,0,107,148C100,147,94,143,94,139C94,134,100,130,107,129'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M143,139C143,134,148,130,156,129A28,28,0,0,0,152,129C142,129,135,133,135,139C135,144,142,149,152,149A28,28,0,0,0,156,148C148,147,143,143,143,139'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M107,129C100,130,94,134,94,139C94,143,100,147,107,148C115,147,120,143,120,139C120,134,115,130,107,129'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M156,129C148,130,143,134,143,139C143,143,148,147,156,148C163,147,169,143,169,139C169,134,163,130,156,129'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M87,105A6,6,0,0,0,83,116A6,6,0,0,0,90,118C96,116,108,115,117,128C117,128,115,101,87,105'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M168,105C140,101,138,128,138,128C147,115,159,116,165,118A6,6,0,0,0,172,116A6,6,0,0,0,168,105'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M107,135A3,3,0,0,0,104,139A3,3,0,0,0,107,142A3,3,0,0,0,111,139A3,3,0,0,0,107,135'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M156,135A3,3,0,0,0,152,139A3,3,0,0,0,156,142A3,3,0,0,0,159,139A3,3,0,0,0,156,135'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M98,87C112,87,109,98,124,95C140,93,145,81,157,91'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M110,101C118,101,116,107,126,106C135,104,138,97,145,103'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M80,139A23,15,0,1,0,126,139A23,15,0,1,0,80,139'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M129,139A23,15,0,1,0,175,139A23,15,0,1,0,129,139'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M202,153A5,5,0,0,0,197,159A5,5,0,1,0,202,153'%20fill='%23ABABAB'%20stroke='%23000'/%3e%3cpath%20d='M53,153A5,5,0,0,0,47,159A5,5,0,1,0,53,153'%20fill='%23ABABAB'%20stroke='%23000'/%3e%3cpath%20d='M203,154A5,5,0,0,0,199,155V155A5,5,0,0,1,204,160A4,4,0,0,1,203,164A5,5,0,1,0,203,154V154'%20fill='%23777777'/%3e%3cpath%20d='M47,159A5,5,0,0,0,52,164A4,4,0,0,1,51,160A5,5,0,0,1,56,155V155A4,4,0,0,0,52,154A5,5,0,0,0,47,159'%20fill='%23777777'/%3e%3cpath%20d='M173,217H82C54,224,48,255,48,255H207C207,255,201,224,173,217'%20fill='%23060508'%20stroke='%23000'/%3e%3cpath%20d='M82,211H173V255H82V211'%20fill='%23222033'%20stroke='%23000'/%3e%3cpath%20d='M82,211V225C95,238,111,246,127,246C144,246,160,238,173,225V211H82'%20fill='%23000000'/%3e%3cpath%20d='M193,144H62A6,6,0,0,0,55,152C63,199,92,235,127,235C163,235,192,199,199,152A6,6,0,0,0,193,144'%20fill='%23222033'%20stroke='%23000'/%3e%3cpath%20d='M193,144H153C146,184,124,216,96,225C105,231,116,235,127,235C163,235,192,199,199,152A6,6,0,0,0,193,144'%20fill='%23000000'/%3e%3cpath%20d='M166,42A42,42,0,0,1,162,55C171,61,197,94,197,125C197,125,211,46,166,42'%20fill='%23513E3A'%20stroke='%23000'/%3e%3cpath%20d='M61,84C33,124,61,179,61,179C61,221,96,242,96,242C82,145,96,90,96,90C104,91,112,87,113,76C143,80,156,66,162,55A42,42,0,0,0,166,42C103,0,56,60,61,84'%20fill='%23764B42'%20stroke='%23000'/%3e%3cpath%20d='M139,32A66,66,0,0,1,149,33A68,68,0,0,0,123,28C104,28,86,37,73,52C64,63,60,76,61,84V84V84C33,124,61,178,61,179V179V179C61,216,90,237,95,240C95,237,94,234,94,231C83,217,77,202,77,185C76,182,50,129,77,90C75,81,80,68,89,57C102,42,120,32,139,32'%20fill='%23472620'/%3e%3cpath%20d='M187,54A28,28,0,0,0,166,43A46,46,0,0,1,164,51C187,55,193,82,193,103A79,79,0,0,1,197,120C199,107,201,72,187,54'%20fill='%2330211F'/%3e%3cstyle%3epath{stroke-width:0.71}%3c/style%3e%3c/svg%3e", "name": "Dreamer #9", "description": "Runners run, but sometimes they dream. This is one of their dreams.", "attributes": ""}